rs/README.md

```diff
@@ -27,4 +27,22 @@ fn main() {
     println!("max: {}", output.max);
     println!("min: {}", output.min);
 }
-```
\ No newline at end of file
+```
+
+If you would like to supply a random number generator:
+
+```rust
+use ds_heightmap::Runner;
+use rand_chacha;
+
+fn main() {
+    let mut rng = ChaCha8Rng::from_entropy();
```

Comment:
1. `ChaCha8Rng` is undeclared here.
2. `from_entropy` not found in `ChaCha8Rng`.

This example is not runnable.